### DISCUSSION TITLE
Trans Alt

### DISCUSSION THREAD

**Discussion by LukasGNN** [flybywiresim/aircraft#3698] [Help] [answered]
Hi,

In latest DEV version there is no way to set up Trans Alt i Perf (Take Off). 
The field i blank.
Anyone experiencing the same?

BR
Lukas

**Comment by hanomax24** [comment]
I just updated to the latest DEV version. No problems with inserting the transition altitude on the perf page. Did you fill out the init page in advance?

**Comment by LukasGNN** [comment, marked as answer]
Nope, but in meantime I found the problem. Or at least the reason.
If there is no valid flight plan loaded into MCDU the Trans Att is empty and cannot be inserted. It must be valid flight plan loaded:)

**Comment by hanomax24** [comment]
You can load a flight plan or fill out the init page in advance.

**Reply by LukasGNN** [reply]
i see, how to fiil out the ini in advance?

**Comment by hanomax24** [comment]
After going to the INIT A page and setting the FROM / TO airports you can insert the transition altitude.
